### ISSUE TITLE
Version fetch from .git-based Plugin

### ISSUE THREAD

**Issue by bennybennet** [SamVerschueren/gulp-cordova-plugin#10]
With Version 1.3.0 ist it possible to add a specific version for a plugin :+1: . 
There's a bug when adding .git-based Plugins. like so:

```
{
    "https://github.com/litehelpers/Cordova-sqlcipher-adapter": "latest",
    "com.ionic.keyboard": "latest",
    "https://github.com/kristianhristov/cordova-cookie-master.git": "latest"
}
```

```
fatal: unable to access 'https://github.com/litehelpers/Cordova-sqlcipher-adapter@latest/': The requested URL returned error: 400
```

**Comment by bennybennet** [comment]
I can use 

```
"https://github.com/kristianhristov/cordova-cookie-master.git": true
```

as a workaround, but that's not working for 

```
"https://github.com/dubcanada/phonegap-googlemaps-plugin.git": {
      "version": true,
      "variables": {
        "API_KEY_FOR_ANDROID": "YOUR_ANDROID_API_KEY_IS_HERE",
        "API_KEY_FOR_IOS": "YOUR_IOS_API_KEY_IS_HERE"
      }
    }
```

```
fatal: unable to access 'https://github.com/dubcanada/phonegap-googlemaps-plugin.git@true/': The requested URL returned error: 400
```

**Comment by SamVerschueren** [comment]
I will have a look at it, thanks for replying. The version property is optional so just don't provide it in your last example.

**Comment by SamVerschueren** [comment]
This is implemented in `v1.3.1`. When a version is provided, it adds `#v1.2.0` at the end of the git url.

``` json
"https://github.com/dubcanada/phonegap-googlemaps-plugin.git": {
    "version": "1.2.0",
    "variables": {
        "API_KEY_FOR_ANDROID": "YOUR_ANDROID_API_KEY_IS_HERE",
        "API_KEY_FOR_IOS": "YOUR_IOS_API_KEY_IS_HERE"
    }
}
```

`https://github.com/dubcanada/phonegap-googlemaps-plugin.git#v1.2.0`
